airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/metadata.py

```diff
@@ -100,79 +100,6 @@ def compute_registry_overrides(merged_df):
     return registries
 
 
-def merge_into_metadata_definitions(
```

Comment:
This was an artifact from when we transformed the old registry into the metadata files at the very begining.

Because we no longer do that, this function is unused, it just happened to be missed by the grim reaper last time